.gitignore

```diff
@@ -69,3 +69,8 @@ opera-2017-multiplicity-studies/outputs/opera-events.json
 opera-2017-multiplicity-studies/outputs/opera-events.json
 opera-2019-electron-neutrinos/outputs/opera-events.json
 opera-2019-neutrino-induced-charm/outputs/opera-events.json
+opera-2019-electron-neutrinos/outputs/opera-events.json
+cms-2016-collision-datasets/inputs/config-store
```

Comment:
(1) If we put config files into `gitignore`, then we should remove them from the PR.

Is it OK to squash your pull request and remove all the config store files?

(2) The same comment applies to `outputs/*.json` which are also in gitignore but are existing in the PR.

(3) Shall we do the same with `hlt-config-files`? Keep them in inputs and commit, or not commit but then add into `.gitignore`?